python/MOCK_AIS.py

```diff
@@ -146,6 +167,47 @@ def gps_callback(self, msg):
     def speedup_callback(self, msg):
         self.speedup = msg.data
 
+    def get_real_ships(self):
+        lat = self.sailbot_lat
+        lon = self.sailbot_lon
+        space = "%20"
+        pos_list = ""
+        pos_list += str(lon-0.2) + space + str(lat-0.2) + space # bot left
```

Comment:
This is a great start, but may be best to use a fixed distance like 30km x 30km square, then convert to lat/lon using our utilities functions